packages/turf-transform-scale/index.js

```diff
@@ -0,0 +1,71 @@
+var bbox = require('@turf/bbox');
+var meta = require('@turf/meta');
+var helpers = require('@turf/helpers');
+var centroid = require('@turf/centroid');
+var invariant = require('@turf/invariant');
+var rhumbBearing = require('@turf/rhumb-bearing');
+var rhumbDistance = require('@turf/rhumb-distance');
+var rhumbDestination = require('@turf/rhumb-destination');
+var point = helpers.point;
+var coordEach = meta.coordEach;
+var getCoords = invariant.getCoords;
+
+
+/**
+ * Moves any geojson Feature or Geometry of a specified distance along a Rhumb Line
+ * on the provided direction angle.
+ *
+ * @name scale
+ * @param {GeoJSON} geojson object to be scaled
+ * @param {number} factor of scaling, positive values greater than 0
+ * @param {boolean} [fromCenter=true] point from which the scaling will occur; it will default to the most
```

Comment:
👍 Love it!

We might need to re-word the JSDocs, sounds like the "default" is saying `False` but in fact it's `True`.

> point from which the scaling will occur; **`it will default`** to the most south-west point of a `bbox` containing the geojson, if true the scaling will origin from the `centroid` of the geojson